### ISSUE TITLE
לא ברור למה בשלב 2 הניקוד של החים מתחיל לרדת

### ISSUE THREAD

**Issue by VictoKu1** [GameDev-Tommy-Bar/window-cleaner#2]
אולי כדאי להחליף את זה לכסף, שיהיה יותר ברור ?

**Comment by VictoKu1** [comment]
כשמנסים להוסיף נקודות ל Dirt Slower שום דבר לא מתשנה.

**Comment by VictoKu1** [comment]
אין שלב 3.

**Comment by VictoKu1** [comment]
בעמוד של ה Stats, אולי כדאי לקצר את המספרים, וגם לא ברור למה הכפתור  Add points נמצא שם.

**Comment by tommyg141** [comment]
שלב 3\4\5\6 יעלו עוד מעט, לגבי הadd points זה היה בשבילנו כתבנו להוריד תודה
לגבי dirt נבדוק 
אתה מתכוון את הנקודות בצד ? לגבי הכסף

**Comment by VictoKu1** [comment]
כן, מה שיורד כל כמה שניות בלי סיבה, כשזה הניקוד של החיים לא ממש ברור למה זה ככה ולמה הוספת כסף מתקנת את זה.

**Comment by tommyg141** [comment]
אתה מתכוון למד חיים של הבוס ? 
זה כי אם לא תסיים לנקות ימשיך לרדת הרמת עצבים שלו , כי בסוף הנקיון אתה צריך לאסוף את המטבע ואז זה שוב מתלכלך

**Comment by VictoKu1** [comment]
עדיף לשנות את התמונה למטבעות.

**Comment by VictoKu1** [comment]
עוד משהו קטן, המיקום של התמונה של הקסדה זה משהו שכדאי לישר.

**Comment by tommyg141** [comment]
עוד מעט נעלה את המשחק החדש, נגיב פה כדי שתוכל לראות
בגדול אנחנו נוסיף כמו מטבע ליד הניקוד ואז יהיה יותר ברור

**Comment by bar-schtalman** [comment]
> כשמנסים להוסיף נקודות ל Dirt Slower שום דבר לא מתשנה.
ידוע וטופל, היינו צריכים לבצע כמה שינויים כדי להכניס באמת את השיפור הזה למשחק בצורה טובה, בקרוב מעלים גרסה מתוקנת

**Comment by bar-schtalman** [comment]
המשחק עודכן
https://tommy-bar.itch.io/windows-cleaner
